mypy/join.py

```diff
@@ -66,13 +66,17 @@ def join_types(s: Type, t: Type) -> Type:
     if isinstance(s, ErasedType):
         return t
 
+    if isinstance(s, UnionType) and not isinstance(t, UnionType):
+        s, t = t, s
+
     if isinstance(s, NoneTyp) and not isinstance(t, NoneTyp):
         s, t = t, s
 
     # Use a visitor to handle non-trivial cases.
     return t.accept(TypeJoinVisitor(s))
 
 
+
```

Comment:
Oops!  That was accidental.  As they say, "0.002K blank lines ought to be enough for anybody".